From 07a2b332512099051c979800095d6e2e7f3afe80 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 13:12:39 -0400
Subject: [ethernet] i40evf: fix typo

Message-id: <1409663591-32531-117-git-send-email-sassmann@redhat.com>
Patchwork-id: 90596
O-Subject: [RHEL7.1 PATCH 116/148] i40evf: fix typo
Bugzilla: 1091127
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Mitch Williams <mitch.a.williams@intel.com>

Correct a missing word in a log message.

Change-ID: Id94da7d9f842382d073b3947e0b616503e2f8e91
Signed-off-by: Mitch Williams <mitch.a.williams@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 3f2ab1721f77935eb4ee50b78c6ecfe24b3a2648)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40evf/i40evf_main.c b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
index 36b1ad7..7138ab4 100644
--- a/drivers/net/ethernet/intel/i40evf/i40evf_main.c
+++ b/drivers/net/ethernet/intel/i40evf/i40evf_main.c
@@ -2022,7 +2022,7 @@ static void i40evf_init_task(struct work_struct *work)
   }
   err = i40evf_send_vf_config_msg(adapter);
   if (err) {
-   dev_err(&pdev->dev, "Unable send config request (%d)\n",
+   dev_err(&pdev->dev, "Unable to send config request (%d)\n",
     err);
    goto err;
   }
-- 
1.7.1